src/app/(markets)/markets/[address]/page.tsx

```
@@ -483,172 +259,175 @@ const MarketPage = ({ params }: { params: { address: Address } }) => {
+            {fetchMarketLoading ? (
+              <Box px={isMobile ? '16px' : 0}>
+                <Skeleton height={120} />
+              </Box>
+            ) : (
+              <Tabs position='relative' variant='common' mx={isMobile ? '16px' : 0}>
+                <TabList>
+                  {tabs.map((tab) => (
+                    <Tab key={tab.title}>
+                      <HStack gap={isMobile ? '8px' : '4px'} w='fit-content'>
+                        {tab.icon}
+                        <>{tab.title}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)